bin/dockerfilelint

```diff
@@ -4,21 +4,25 @@ var process = require('process');
 var fs = require('fs');
 var os = require("os");
 var path = require('path');
-var argv = require('yargs').argv;
+var argv = require('yargs')
+            .alias('o', 'output')
+            .describe('output', 'the format in which you want to display the lint results (default is cli)')
```

Comment:
(Sorry, this was supposed to be part of the review.)

I kinda think a boolean CLI option like `--json` or `-j` would be better (easier to use/remember) than `-o json`. We could even support both.

Might I suggest the following yargs config code:

```js
var argv = require('yargs')
  .usage('Usage: dockerfilelint [files | content..] [options]')
  .option('o', {
    alias: 'output',
    desc: 'Specify the format to use for output of linting results. Valid values are `json` or `cli` (default).',
    type: 'string'
  })
  .option('j', {
    alias: 'json',
    desc: 'Output linting results as JSON, equivalent to `-o json`.',
    type: 'boolean'
  })
  .help().alias('h', 'help')
  .example('dockerfilelint Dockerfile', 'Lint a Dockerfile in the current working directory\n')
  .example('dockerfilelint test/example/* -j', 'Lint all files in the test/example directory and output results in JSON\n')
  .example(`dockerfilelint 'FROM latest'`, 'Lint the contents given as a string on the command line\n')
  .example('dockerfilelint < Dockerfile', 'Lint the contents of Dockerfile via stdin')
  .wrap(85)
  .check(argv => {
    if (!argv.output && argv.json) argv.output = 'json'
    return true
  })
  .argv;
```